### ISSUE TITLE
Add link name for home button (logo in upper left)

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2528]
**OCIO Violation:** Ensure link text is meaningful within context

**OCIO Note:** This element does not have a mechanism that allows an accessible name value to be calculated

**OCIO priority:** 7.5

More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=bLTMhX) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/login.php)

**Instances**

- [ ] Home button - logo in upper left.  Since the URL in the code for this is just “/” - that’s not clear where it’s goes to. 

The OCIO violation brings up 4 different code snippets causing this:

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/3f0ad1b6-2a0a-4640-acbe-0c31a68f3ae3)

**Comment by greensomes** [comment]
I added the aria-label attribute with the value "Link to home page" to the logo anchor tag. Please see screenshot below.

![Screenshot 2023-08-15 at 11.41.01 AM.png](https://images.zenhubusercontent.com/6361168797f5f7285b87509f/887084ac-aa2c-48fc-b43a-6edf0781846c)

**Comment by Maroyafaied** [comment]
@greensomes Did you make this change in the PR that's open for August data or is it in it's own branch?

**Comment by eelzi-ONRR** [comment]
https://nrrd-preview.app.cloud.gov/sites/2528-add-link-name-for-home-button-logo/

I checked the preview in ANDI and it reads correctly [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBa0VJIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--ca3d0d2cc8c0c81bd92ae00bfe6a02a899693a16/image.png)
